modules/trusted-firmware-m/Kconfig.tfm.pm

```
@@ -28,14 +28,14 @@ config PM_PARTITION_SIZE_TFM
 	default 0xFE00 if TFM_PROFILE_TYPE_MINIMAL && TFM_CMAKE_BUILD_TYPE_DEBUG && \
 		BOOTLOADER_MCUBOOT
 	default 0x10000 if TFM_PROFILE_TYPE_MINIMAL && TFM_CMAKE_BUILD_TYPE_DEBUG
-	default 0x6800 if TFM_PROFILE_TYPE_MINIMAL && BOOTLOADER_MCUBOOT && SOC_NRF54L15_ENGA_CPUAPP
+	default 0x6800 if TFM_PROFILE_TYPE_MINIMAL && BOOTLOADER_MCUBOOT && (SOC_NRF54L15_ENGA_CPUAPP || SOC_NRF54L15_ENGA_CPUAPP)
```

Did you mean:

```suggestion
	default 0x6800 if TFM_PROFILE_TYPE_MINIMAL && BOOTLOADER_MCUBOOT && (SOC_NRF54L15_ENGA_CPUAPP || SOC_NRF54L15_CPUAPP)
```

Same below?